atc/creds/pool.go

```diff
@@ -61,11 +63,9 @@ func (pool *varSourcePool) FindOrCreate(logger lager.Logger, config map[string]i
 		pool.pool[key] = &inPoolManager{
 			manager:        manager,
 			secretsFactory: secretsFactory,
-			lastUseTime:    time.Now(),
```

Comment:
Why was this removed now? :thinking:

Reply:
Because `lastUseTime` is updated in `inPoolManager.NewSecrets()`.

Reply:
Ah got it, thanks!